### PR TITLE
Add instructions on assigning an environment tag

### PR DESCRIPTION
**Pull request by erjohnson**
Added instructions on assigning an environment tag to Client Overview doc.

[Clients, Assign an Environment Tag](https://auth0-docs-content-pr-5037.herokuapp.com/docs/clients#assign-an-environment-tag)

### CODE REVIEW

File: articles/clients/index.md [hunk 1]
```diff
@@ -46,6 +46,22 @@ A new client will be created and you will be redirected to this client's view th
 
 - [Connections](${manage_url}/#/clients/${account.clientId}/connections): Connections are sources of users. They are categorized into Database, Social and Enterprise and can be shared among different clients. For more details refer to: [Connections](/clients/connections). For a detailed list on the supported Identity Providers refer to: [Identity Providers Supported by Auth0](/identityproviders).
 
+### Assign an Environment Tag
+
+After creating a client, make sure to assign environment tags to your tenants to differentiate between development, staging, and production environments.
+
+To assign an environment tag to a tenant, go to the [Tenants](https://support.auth0.com/tenants/public) page on the [Auth0 Support Center](https://support.auth0.com/). Locate your tenant and click the gear icon to bring up the **Settings** section.
```
Comment (mpaktiti) [inline articles/clients/index.md]: Use `${env.DOMAIN_URL_SUPPORT}` instead of `https://support.auth0.com/`

File: articles/clients/index.md [hunk 2]
```diff
@@ -46,6 +46,22 @@ A new client will be created and you will be redirected to this client's view th
 
 - [Connections](${manage_url}/#/clients/${account.clientId}/connections): Connections are sources of users. They are categorized into Database, Social and Enterprise and can be shared among different clients. For more details refer to: [Connections](/clients/connections). For a detailed list on the supported Identity Providers refer to: [Identity Providers Supported by Auth0](/identityproviders).
 
+### Assign an Environment Tag
+
+After creating a client, make sure to assign environment tags to your tenants to differentiate between development, staging, and production environments.
```
Comment (mpaktiti) [inline articles/clients/index.md]: This is not a mandatory step each time you create a new client, so I would rephrase `make sure to assign environment tags to your tenants` to sth like make sure that your tenant has the proper environment set.

Comment (mpaktiti) [reply articles/clients/index.md]: I would also add a note about setting up multiple environments and how the info can be found here https://auth0.com/docs/dev-lifecycle/setting-up-env

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Left some comments for some minor changes. I would also update this doc https://auth0.com/docs/dev-lifecycle/setting-up-env and add a note saying how for each new tenant they create they have to specify also the environment.

**Comment by erjohnson** [comment]
I moved the how-to section to the Set Up Multiple Environments page, since it seems to fit more there. I can move it back if needed 😄

File: articles/clients/index.md [hunk 3]
```diff
@@ -46,6 +46,14 @@ A new client will be created and you will be redirected to this client's view th
 
 - [Connections](${manage_url}/#/clients/${account.clientId}/connections): Connections are sources of users. They are categorized into Database, Social and Enterprise and can be shared among different clients. For more details refer to: [Connections](/clients/connections). For a detailed list on the supported Identity Providers refer to: [Identity Providers Supported by Auth0](/identityproviders).
 
+### Check the Tenant Environment
```
Comment (mpaktiti) [inline articles/clients/index.md]: This paragraph is too much now. Replace it with only a note like this one:

```
::: note
After you create your first client, set the environment for your tenant: development, staging, or production. For more information refer to [Set Up Multiple Environments](/dev-lifecycle/setting-up-env#environment-tags).
:::
```

File: articles/dev-lifecycle/setting-up-env.md [hunk 4]
```diff
@@ -13,6 +13,22 @@ The example above uses a simple naming convention to distinguish each environmen
 You can request a [child account](/dev-lifecycle/child-tenants) for use in a development/staging/testing environment. Note that free accounts do not include a child account.
 :::
 
+## Environment Tags
```
Comment (mpaktiti) [inline articles/dev-lifecycle/setting-up-env.md]: Rephrase to `Set the Environment` (the link will change, so update any links pointing here)

File: articles/dev-lifecycle/setting-up-env.md [hunk 5]
```diff
@@ -13,6 +13,22 @@ The example above uses a simple naming convention to distinguish each environmen
 You can request a [child account](/dev-lifecycle/child-tenants) for use in a development/staging/testing environment. Note that free accounts do not include a child account.
 :::
 
+## Environment Tags
+
+For each new tenant created, you should specify its environment. You can assign environment tags to your tenants to differentiate between development, staging, and production environments.
+
+To assign an environment tag to a tenant, go to the [Tenants](${env.DOMAIN_URL_SUPPORT}tenants/public) page on the [Auth0 Support Center](${env.DOMAIN_URL_SUPPORT}). Locate your tenant and click the gear icon to bring up the **Settings** section.
```
Comment (mpaktiti) [inline articles/dev-lifecycle/setting-up-env.md]: `${env.DOMAIN_URL_SUPPORT}tenants/public` should be `${env.DOMAIN_URL_SUPPORT}/tenants/public`

Comment (mpaktiti) [reply articles/dev-lifecycle/setting-up-env.md]: Also I would rephrase this to: `go to the [Auth0 Support Center > Tenants](${env.DOMAIN_URL_SUPPORT}/tenants/public).`

File: articles/dev-lifecycle/setting-up-env.md [hunk 6]
```diff
@@ -13,6 +13,22 @@ The example above uses a simple naming convention to distinguish each environmen
...
 
+## Environment Tags
+
+For each new tenant created, you should specify its environment. You can assign environment tags to your tenants to differentiate between development, staging, and production environments.
+
+To assign an environment tag to a tenant, go to the [Tenants](${env.DOMAIN_URL_SUPPORT}tenants/public) page on the [Auth0 Support Center](${env.DOMAIN_URL_SUPPORT}). Locate your tenant and click the gear icon to bring up the **Settings** section.
+
+![Support Center Tenants](/media/articles/clients/support-tenants.png)
+
+Next, select the **Assign Environment Tag** option. Use the form to identify your tenant's environment as either **Development**, **Staging**, or **Production**.
```
Comment (mpaktiti) [inline articles/dev-lifecycle/setting-up-env.md]: Use bold for field names and code marking for values: `Development`, `Staging`, or `Production`

File: articles/dev-lifecycle/setting-up-env.md [hunk 7]
```diff
@@ -13,6 +13,22 @@ The example above uses a simple naming convention to distinguish each environmen
...
+
+If your tenant is mixed use, choose the higher environment. For example, a tenant used for both development and production should be set to **Production**.
```
Comment (mpaktiti) [inline articles/dev-lifecycle/setting-up-env.md]: `Production`

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Some more small changes

**Comment by mpaktiti** [comment]
Looks good 👍
